### ISSUE TITLE
soho standalone pager component: accessibility issues

### ISSUE THREAD

**Issue by JBismonte** [infor-design/enterprise-ng#994]
**Describe the bug**
There are two accessibility issues found in the soho-standalone-pager component:
1. The page size button element with aria-haspopup="true" cannot be activated with the Down Arrow or Up Arrow.
2. The soho page buttons appear to use a roving tabindex rather than individually receiving keyboard focus with the tab key. There is no role or visual indication for this functionality. 

**To Reproduce**
on your local enterprise-ng
1. run npm start
2. Go to http://localhost:4200/ids-enterprise-ng-demo/datagrid-standalone-pager
3. navigate to the pager buttons via keyboard, you cannot navigate from one button to another button via tab, but you can with arrow keys.
4. also when you navigate to the select page size button via keyboard, clicking down doesn't activate the button.

**Expected behavior**
[VPAT Recommendations]
1. regarding the page size button: Allow Down Arrow and Up Arrow functionality to open the menu. The Down Arrow should function similarly to the Enter or Spacebar. The Up Arrow should open the menu and focus on the last menu item. 
2. regarding the first, previous, next and last buttons: Remove the arrow key functionality and ensure that each of these controls can receive focus with the Tab key instead. 

**Version**
- ids-enterprise-ng: 9.2.0-dev

**Platform**
 - Device (if applicable) :NA
 - OS Version: Windows 10
 - Browser Name: Firefox
 - Browser Version: v85.0

**Additional context**
https://jira.lawson.com/browse/LMCLIENT-34407
